rosdep/base.yaml

```
@@ -2979,6 +2979,17 @@ libconsole-bridge-dev:
 libcos4-dev:
   debian: [libcos4-dev]
   ubuntu: [libcos4-dev]
+libcpp-httplib-dev:
+  debian:
+    bookworm: [libcpp-httplib-dev]
+    sid: [libcpp-httplib-dev]
```

To make this futureproof, we should probably do:

```suggestion
  debian:
    '*': [libcpp-httplib-dev]
    bullseye: null
```